From 1edd503003c1de78ed15a81f882440c4d0a8f2be Mon Sep 17 00:00:00 2001
From: Mohan Kumar G <quic_mkumarg@quicinc.com>
Date: Thu, 2 Jan 2025 12:55:50 +0530
Subject: [PATCH] QSDK:CP: wifi-scripts: Fix syntax error in DFS puncture
 config

Currently, DFS puncturing config is not added to config file
due to syntax error.

Fix this issue by adding correct syntax for DFS puncture config

Change-Id: Ia6e1adf1b65dec8ac6a1e54544ebbb1cd362d018
Signed-off-by: Mohan Kumar G <quic_mkumarg@quicinc.com>
---
 .../config/wifi-scripts/files/lib/netifd/wireless/mac80211.sh   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/network/config/wifi-scripts/files/lib/netifd/wireless/mac80211.sh b/package/network/config/wifi-scripts/files/lib/netifd/wireless/mac80211.sh
index fb17097f8a32..1d9a0eae3861 100755
--- a/package/network/config/wifi-scripts/files/lib/netifd/wireless/mac80211.sh
+++ b/package/network/config/wifi-scripts/files/lib/netifd/wireless/mac80211.sh
@@ -764,7 +764,7 @@ mac80211_hostapd_setup_base() {
 				append base_cfg "ru_punct_acs_threshold=$ru_punct_acs_threshold" "$N"
 			fi
 
-			[-n "$use_ru_puncture_dfs" ] && append base_cfg "use_ru_puncture_dfs=$use_ru_puncture_dfs" "$N"
+			[ -n "$use_ru_puncture_dfs" ] && append base_cfg "use_ru_puncture_dfs=$use_ru_puncture_dfs" "$N"
 		fi
 
 		if [ "$band" = "6g" ]; then
-- 
2.34.1

